src/main/resources/META-INF/reaktivity/http.idl

```diff
@@ -36,5 +36,12 @@ scope http
         {
             list<http::HttpHeader> headers;
         }
+
+        struct HttpDataEx
```

Comment:
This looks like core::stream::Data. So is this necessary ? or extend core::stream::Data